packages/phrasebook/src/locale/en/card.ts

```diff
@@ -0,0 +1,10 @@
+import { Phrasebook } from '../../translation.js';
+import './shared.js';
+
+const translations = {
+  MENU: 'menu'
```

Comment:
Name is as OPEN_MENU: 'Open Menu'

Reply:
I think `menu` is more generic than `Open Menu`

Reply:
It is more generic, but the button opens menu, is not it? :)

Reply:
It is OK for both. Action Menu is a new option which one you prefer?

Reply:
I still prefer `Open Menu` :)

Reply:
Updated